truckersmp_cli/main.py

```diff
@@ -222,7 +222,9 @@ def start_with_proton():
         logging.info("Using Steam Runtime container")
         # share directories with Steam Runtime container
         shared_paths = [Args.gamedir, Args.protondir, Args.prefixdir]
-        if not Args.singleplayer:
+        if Args.singleplayer:
+            shared_paths += get_steam_library_dirs(steamdir)
```

Comment:
```suggestion
        if Args.singleplayer:
            # workshop mods may be loaded from other steam libraries
            # despite they are also present in our gamedir library
            shared_paths += get_steam_library_dirs(steamdir)
```

Reply:
Feel free to commit the suggestion when the others are ready.